index.js

```diff
@@ -4,7 +4,44 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let args = process.argv.slice(2);
+    let answer;
+    if (args[0] === "plus"){
```

Comment:
If you have your conditional for checking if the operator is `plus` or `minus` in your for loop, you can avoid repeating code.